site/operational-guide.markdown

```diff
@@ -0,0 +1,384 @@
+discovery is enabled by default.
+
+## Network Partition
+
+A network partition is a transient condition whereby some arbitrary
+subsets of peers are unable to communicate with each other. Weave is
+designed to allow peers and their containers to make maximum safe
+progress under conditions of partition, healing automatically once the
+partition is over.
+
+## IP Address Manager (IPAM)
+
+IPAM is the subsystem responsible for dividing up a large contiguous
+block of IP addresses (known as the IP allocation range) amongst peers
+so that individual addresses may be uniquely assigned to containers
+anywhere on the overlay network.
+
+When a new network is formed an initial division of the IP allocation
+range must be made. Two (mutually exclusive) mechanisms with different
+tradeoffs are provided to perform this task: seeding and consensus.
+
+## Seeding
+
+Seeding requires each peer to be told the list of peer names amongst
+which the address space is to be divided initially. There are some
+constraints and consequences:
+
+* Every peer added to the network _must_ receive the same seed list,
+  for all time, or they will not be able to join together to form a
+  single cohesive whole
+* Because the 'product UUID' and 'random value' methods of peer name
+  assignment are unpredictable, the end user must by necessity also
+  specify peer names
+* Even though every peer _must_ receive the same seed, that seed does
+  _not_ have to include every peer in the network, nor does it have to
+  be updated when new peers are added (in fact due to the first
+  constraint above it may not be)
+
+Example configurations are given in the section on deployment
+scenarios.
+
+## Consensus
+
+Alternatively, when a new network is formed for the first time peers
+can be configured to co-ordinate amongst themselves to automatically
+divide up the IP allocation range - this process is known as
+consensus, and is implemented via a leader election algorithm.
+Consensus based bootstrapping requires each peer to be told the total
+number of expected peers (the 'initial peer count') in order to avoid
+small independent groups of peers from electing different leaders
+under conditions of partition.
+
+Example configurations are given in the section on deployment
+scenarios.
+
+## Electors and Observers
+
+When using consensus-based configuration then by default every peer
+included in the network is a candidate for leadership. This does not
+always make sense (for example when adding relatively short lived
+peers to an existing cluster in response to a scale-out event) and so
+an option is provided to start a peer as an _observer_. Such peers
+will not put themselves forward as leaders, but will abide by the
+results of an election amongst elector-peers.
+
+Example configurations are given in the section on deployment
+scenarios.
+
+## Persistence
+
+If you relaunch weave with _exactly_ the same parameters as the
```

Comment:
Simpler now - we always remember unless you do `weave reset` or the peer name changes